deploy/crds/spinnaker_v1alpha1_spinnakerservice_crd.yaml

```diff
@@ -59,10 +59,16 @@ spec:
                             additionalProperties:
                               type: string
                             type: object
+                          port:
+                            format: int32
+                            type: integer
                           type:
                             type: string
                         type: object
                       type: object
+                    port:
```

Comment:
i kind of feel like having the global port isn't really worthwhile, and might add confusion.

What do you think about not including it?

Reply:
Having a global port can help to have all exposed services running on the same public port (if there's no `overrideBaseUrl` set), also I think it makes sense to have `overrides` section replace values in the global section.